### PR TITLE
fix rhs issues

### PR DESCRIPTION
**Pull request by trilopin**
#### Summary
Fix RHS issues from UX feedback:

`When the RHS is open, the respective icon in the header(timeline or info) should stay highlighted.` this was already done, maybe not in the branch @abhijit-singh tested.

- Adjusted colors, border radius for close/back icons.
- Changed behavior for clicking again in icons with RHS already open (toggle)
- Same height forced for all rows in RunInfoOverview

```
Clicking on the header icons should toggle open and close the [RHS](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#rhs) instead of just opening it.

In the Run info RHS, for the overview section, rows seem to have different heights. We can standardize them all to 44px.

In the RHS, change the colour of the close(x) icon to center-channel-text with 56% opacity instead of 100%. Also, the border radius for the icon button should be 4px. Same styling to be applied to the back icon in the Participants RHS view.
```

Demo
https://user-images.githubusercontent.com/4096774/178490681-c16fc35b-3f84-4ef9-abc1-a1c98d5105bb.mp4

#### Ticket Link
Fixes https://mattermost.atlassian.net/jira/software/c/projects/MM/boards/68?modal=detail&selectedIssue=MM-45675

#### Checklist
- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

File: webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx [hunk 1]
```diff
@@ -175,8 +175,8 @@ const PlaybookRunDetails = () => {
                 <Header isRHSOpen={RHS.isOpen}>
                     <RunHeader
                         playbookRun={playbookRun}
-                        onViewInfo={onViewInfo}
-                        onViewTimeline={onViewTimeline}
+                        onViewInfo={RHS.isOpen && RHS.section === RHSContent.RunInfo ? RHS.close : onViewInfo}
+                        onViewTimeline={RHS.isOpen && RHS.section === RHSContent.RunTimeline ? RHS.close : onViewTimeline}
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:179]: Nit: Maybe we can refactor that into a function? Something like `toggleRHSWithSection()`

**Review by agarciamontoro — APPROVED** [review]
Thank you!

File: webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx (hunk 1 above)
Comment (trilopin) [reply webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:179]: yep, maybe even change the name of the prop to `onClickInfo` and `onClickTimeline` to be more accurate

**Review by abhijit-singh — APPROVED** [review]
💯

**Review by calebroseland — COMMENTED** [review]
✓

**Pull request merged**